src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -385,22 +388,112 @@ void getHearingLocations() {
             .dwpIsOfficerAttending("Yes")
             .build();
 
-        given(venueService.getEpimsIdForVenue(caseData.getProcessingVenue())).willReturn(Optional.of("219164"));
+        given(venueService.getEpimsIdForVenue(caseData.getProcessingVenue()))
+            .willReturn(Optional.of(HEARING_VENUE_ID_1));
         given(referenceDataServiceHolder.getVenueService()).willReturn(venueService);
 
         List<HearingLocation> result = HearingsDetailsMapping.getHearingLocations(caseData, referenceDataServiceHolder);
 
-        assertThat(result)
-            .hasSize(1)
-            .extracting("locationId","locationType")
-            .containsExactlyInAnyOrder(tuple("219164", COURT));
+        checkHearingLocationResults(result,
+            new Tuple[] {tuple(HEARING_VENUE_ID_1, COURT)});
     }
 
     @DisplayName("When override Hearing Venue Epims Ids is not empty getHearingLocations returns the override values")
     @Test
     void getHearingLocationsOverride() {
+        SscsCaseData caseData = buildOverrideHearingLocations();
 
-        SscsCaseData caseData = SscsCaseData.builder()
+        List<HearingLocation> result = HearingsDetailsMapping.getHearingLocations(caseData, referenceDataServiceHolder);
+
+        checkHearingLocationResults(result,
+            new Tuple[] {tuple(HEARING_VENUE_ID_1, COURT), tuple(HEARING_VENUE_ID_2, COURT)});
+    }
+
+    @DisplayName("When a case has been adjourned and a different venue has been selected, return the new venue")
+    @Test
+    void getHearingLocationsAdjournmentNewVenue() {
+        SscsCaseData caseData = buildOverrideHearingLocations();
+
+        setupAppeal(caseData);
+        setupAdjournedHearingVenue(caseData, "somewhereElse");
+
+        checkHearingLocationResults(HearingsDetailsMapping.getHearingLocations(caseData, referenceDataServiceHolder),
+            new Tuple[] {tuple(HEARING_VENUE_ID_1, COURT)});
+    }
+
+    @DisplayName("When a case has been adjourned and the same venue has been selected, return the same venue")
+    @Test
+    void getHearingLocationsAdjournmentSameVenue() {
+        SscsCaseData caseData = buildOverrideHearingLocations();
+
+        setupAppeal(caseData);
+        setupAdjournedHearingVenue(caseData, "sameVenue");
+
+        caseData.setHearings(Collections.singletonList(Hearing.builder()
+                    .value(uk.gov.hmcts.reform.sscs.ccd.domain.HearingDetails.builder()
```

Comment:
I think this HearingDetails import can be done at the top of the file?

Reply:
There are 2 classes called HearingDetails, it creates a name conflict when importing them both. Easiest to leave it as so as it's only one ref

Reply:
ok